jobs/build/refresh-images/Jenkinsfile

```diff
@@ -91,6 +91,35 @@ node('openshift-build-1') {
                         error("Version overrides must start with 'v'")
                     }
                     oit_update_docker_args = "--version ${VERSION_OVERRIDE}"
+
+                    // Get the OCP version from the current build
+                    // query-rpm-version returns "version: v3.10.0" for example
+                    def detected_version = buildlib.oit("""
+--working-dir ${OIT_WORKING} --group 'openshift-${OSE_MAJOR}.${OSE_MINOR}'
+--quiet
+images:query-rpm-version
+--repo-type unsigned
+""", [capture: true]).split(' ')[1]
```

Comment:
Recommend `.split(' ').last()` to be a bit less sensitive to changes in format of oit output.